src/SFA.DAS.AssessorService.Application.Api/Validators/ExternalApi/Certificates/GetBatchCertificateValidator.cs

```diff
@@ -26,48 +26,41 @@ public GetBatchCertificateRequestValidator(IStringLocalizer<GetBatchCertificateR
                 {
                     RuleFor(m => m).CustomAsync(async (m, context, cancellation) =>
                     {
-                        // NOTE: Currently we're making the Certificate & ILR record both mandatory
+                        // NOTE: Currently we're making the Certificate & ILR record both mandatory - this is wrong fixing it!
                         var requestedIlr = await ilrRepository.Get(m.Uln, m.StandardCode);
                         var sumbittingEpao = await organisationQueryRepository.GetByUkPrn(m.UkPrn);
+                        var existingCertificate = await certificateRepository.GetCertificate(m.Uln, m.StandardCode);
+                        var createdByCertificate = await certificateRepository.GetCertificateByOrgIdLastname(m.Uln, sumbittingEpao.EndPointAssessorOrganisationId, m.FamilyName);
 
-                        if (requestedIlr is null || !string.Equals(requestedIlr.FamilyName, m.FamilyName, StringComparison.InvariantCultureIgnoreCase))
-                        {
-                            context.AddFailure(new ValidationFailure("Uln", "Cannot find apprentice with the specified Uln, FamilyName & Standard"));
-                        }
-                        else if (sumbittingEpao is null)
+                        if (sumbittingEpao is null)
                         {
                             context.AddFailure(new ValidationFailure("UkPrn", "Specified UKPRN not found"));
                         }
                         else
                         {
                             var providedStandards = await standardService.GetEpaoRegisteredStandards(sumbittingEpao.EndPointAssessorOrganisationId);
```

Comment:
To rejig the code slightly, in the `else` of `if(submittingEpao is null` I would move some of the calls downwards. No point querying for stuff if the submitting Epao is null.
```
var createdByCertificateTask = certificateRepository.GetCertificateByOrgIdLastname(m.Uln, submittingEpao.EndPointAssessorOrganisationId, m.FamilyName);
                            var providedStandardsTask = standardService.GetEpaoRegisteredStandards(submittingEpao.EndPointAssessorOrganisationId);
                            var requestedIlrTask = ilrRepository.Get(m.Uln, m.StandardCode);
                            var existingCertificateTask = certificateRepository.GetCertificate(m.Uln, m.StandardCode);

                            await Task.WhenAll(createdByCertificateTask, providedStandardsTask, requestedIlrTask, existingCertificateTask);

                            var providedStandards = providedStandardsTask.Result;
                            var requestedIlr = requestedIlrTask.Result;
                            var createdByCertificate = createdByCertificateTask.Result;
                            var existingCertificate = existingCertificateTask.Result;```